src/templates/antora/template.ts

```diff
@@ -299,9 +297,19 @@ const template = (root: Api, toPath: string): void => {
     content: adocNav
   });
 
+  if (structure === 'legacy') {
+    indexPage.pages.forEach((namespace) => {
+      addPage({
+        filename: 'api/' + namespace.path + '/index_' + namespace.path + '.adoc',
```

Comment:
This doesn't look correct as the old system never generated a file like this. It should just be `index.adoc` in the correct directory instead.